### PR TITLE
[FIX] mrp_subcontracting_dropshipping: test sale w/ dropship

### PR DESCRIPTION
**Pull request by guva-odoo**
This commit is related to 01fa18b, which introduced an issue :
the test flow needed a mrp.bom instance, which is not a dependency of
stock_dropshipping. This commit move the test to
mrp_subcontracting_dropshipping whiwh depends of mrp.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by guva-odoo** [comment]
@fw-bot ignore

File: addons/mrp_subcontracting_dropshipping/tests/__init__.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 #  -*- coding: utf-8 -*-
 #  Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import test_purchase_subcontracting
+from . import test_purchase_subcontracting, test_sale_dropshipping
```
Comment (Whenrow) [inline addons/mrp_subcontracting_dropshipping/tests/__init__.py:6]: new line

File: addons/mrp_subcontracting_dropshipping/tests/test_sale_dropshipping.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
+from odoo.tests import Form
+from odoo.addons.mrp_subcontracting.tests.common import TestMrpSubcontractingCommon
+
+
+class TestSaleDropshippingFlows(TestMrpSubcontractingCommon):
+    
+    def test_dropship_with_different_uom(self):
```
Comment (Whenrow) [inline addons/mrp_subcontracting_dropshipping/tests/test_sale_dropshipping.py:7]: Could you write a small comment explaining why you write it here

Comment (guva-odoo) [reply addons/mrp_subcontracting_dropshipping/tests/test_sale_dropshipping.py:7]: @Whenrow Done

**Comment by Whenrow** [comment]
Thanks
robodoo r+

**Pull request closed without merging**